Sources/AppcuesKit/Appcues.swift

```diff
@@ -355,7 +351,7 @@ public class Appcues: NSObject {
         storage.userSignature = properties?.removeValue(forKey: "appcues:user_id_signature") as? String
         if userChanged {
             // when the identified user changes from last known value, we must start a new session
-            sessionMonitor.start()
+            sessionMonitor.reset()
```

Comment:
when the user changes, now we clear the session, and the subsequent `identify()` will start a new one